### ISSUE TITLE
TPS reduction with HV connectors.

### ISSUE THREAD

**Issue by DonBruce64** [BluSunrize/ImmersiveEngineering#1871]
Having some odd behavior with IE connectors in 1.7.10-0.7.6.  Specifically, the TPS for HV connectors on my server is quite high.  I'm aware that IE tests the whole network for connections, so I don't expect super-low tick times.  However, the main base on the server has a line from a bank of HV capacitors, the connector from which adds 2ms of tick time.  That's a big jump from 6ms to 8ms idle with one player on the server.

From what I can tell, it's all the connections in the base that hurt me.  Is this just regular behavior for IE?  If so, is there anything I can do to distribute the tick load?  I've got about 10 HV draws and ~50 lights and other LV machines within a 100x100 area.  Profile info is attached.
![unknown](https://cloud.githubusercontent.com/assets/15241611/22381046/8edeb83c-e485-11e6-98e6-e867c555c6e8.png)

**Comment by chaos234** [comment]
@DonBruce64 - DId you have tried out the 0.7.7 version for IE-1.7.10? If not, can you please try it out and then run again the cmd you have used to get this values?

I don't know if this exists in the latest version but testing it shouldn't hurt ;) because I can't see any TPS Drops with 0.7.7 using /cofh tps in the chat to let me show the current TPS/ms for each world.

**Comment by Bitterholz** [comment]
IE for Minecraft Version 1.7.X are unsupported. Its all 1.10.2 now.

Issues like these are likely fixed in or by the 1.10 versions.

But if you dont mind, recreate your Test setup in 1.10 and watch if the Issue prevails, that would indeed be helpfull.

**Comment by DonBruce64** [comment]
@chaos234 I wasn't running 0.7.7 when I got those numbers, but I am now and I don't notice much change.  I found that the large number of calls was due to me having 1 HV connector feeding 8 Buildcraft conduits, 3 HV transformers to LV, 3 banks of Open Computers units, and about 60 lights.  I separated the networks by using capacitors, and while the lights flicker while the caps charge, I'm not getting 9000 calls per tick.

I can say that the last few times I've run TPS we didn't have issues, but due to people being busy I haven't been able to load the server yet to force the same conditions as when I reported the bug.

@Bitterholz Normally I wouldn't mind, but the issue is on a server with a size of about 10GB.  My internet would die if I tried to download that.  Due to me changing the wiring system I doubt the issue would prevail in 1.10 anyways as it's mostly stopped in 1.7.10.  I'll just have to watch the profiler when we get lag spikes.  We have been getting lag, but I think it's a combination of overuse of IE components with high calls, combined with Archimedes Ships and lots of block updates.  Just wanted to report the one on your end.
